webview/__init__.py

```diff
@@ -76,15 +77,9 @@ def _create_children(other_windows):
 
     _debug = debug
     _user_agent = user_agent
-    #_multiprocessing = multiprocessing
-    multiprocessing = False # TODO
+    _multiprocessing = block
```

Comment:
This variable can be deleted.

Reply:
multiprocessing is a global variable
it is used by window objects

Reply:
It is not used in any meaningful way in window.py, so it can be purged from there as well